doc/lua_api.txt

```diff
@@ -4010,7 +4012,9 @@ Definition tables
         ^ node placement (i.e. schematics and VoxelManip) or air nodes. If
         ^ return true the node is not flooded, but on_flood callback will
         ^ most likely be called over and over again every liquid update
-        ^ interval. Default: nil ]]
+        ^ interval. Note: liquid-type nodes cannot themselves be floodable, else
```

Comment:
I mean that my suggested text should be on it' own line instead of continuing from '^ interval.'.